sidebar.php

```diff
@@ -4,10 +4,10 @@
  *
  * If no active widgets in sidebar, let's hide it completely.
  *
- * @package WordPress
- * @subpackage Twenty_Twelve
+ * @package MIT Libraries News
```

Comment:
This is a small thing, but I'm noticing that some files are using "MITLibraries-news" as their package value, while others are using "MIT Libraries New". The spec for package tags requires that this be a single word, without spaces:

https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.package.pkg.html

Given this, the proper value should be "MITLibraries-news"